From: Robert Schwebel <r.schwebel@pengutronix.de>
Subject: cfgsh: use ncurses

When compiling in ptxdist, we break with this error message:

arm-iwmmx-linux-gnueabi/bin/ld:
cannot find -lcurses
collect2: ld returned 1 exit status
make[1]: *** [cfgsh] Error 1

While there is probably a way to make it work with curses, let's go the
ncurese way for now.

Probably not for upstream.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 Makefile |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: cfgsh-1.1.2/Makefile
===================================================================
--- cfgsh-1.1.2.orig/Makefile
+++ cfgsh-1.1.2/Makefile
@@ -17,8 +17,8 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111 USA.
 
-CFLAGS = -g -Wall
-LDFLAGS = -lcurses -lreadline -lpthread -lrt 
+CFLAGS := -g -Wall
+LDFLAGS := $(LDFLAGS) -lncurses -lreadline -lpthread -lrt
 
 EXECUTABLES = cfgsh
 
